tests/zfs-tests/tests/functional/cli_root/zfs_receive/receive-o-x_props_override.ksh

```diff
@@ -15,46 +15,34 @@
 #
 
 #
-# Copyright 2016, loli10K. All rights reserved.
+# Copyright 2017, loli10K <ezomori.nozomu@gmail.com>. All rights reserved.
 #
 
 . $STF_SUITE/include/libtest.shlib
 . $STF_SUITE/tests/functional/cli_root/zfs_set/zfs_set_common.kshlib
 
 #
 # DESCRIPTION:
-#	Verify ZFS property override options works when receiving a send stream
+# Verify ZFS property override (-o) and exclude (-x) options work when
+# receiving a send stream
 #
 # STRATEGY:
-#	1. Create a filesystem with children.
-#	2. Snapshot the filesystems.
-#	3. Create a full stream without properties.
-#	4. Create also some incremental replication streams with properties.
-#	5. Receive the send streams and verify we can override properties.
+# 1. Create a filesystem with children.
+# 2. Snapshot the filesystems.
+# 3. Create various send streams (full, incremental, replication) and verify
+#    we can both override and exclude native and user properties.
 #
 
 verify_runnable "both"
 
-orig=$TESTPOOL/$TESTFS1
-origsub=$orig/sub
-dest=$TESTPOOL/$TESTFS2
-destsub=$dest/sub
-typeset userprop=$(valid_user_property 8)
-typeset userval=$(user_property_value 8)
-typeset streamfile_full=/var/tmp/streamfile_full.$$
-typeset streamfile_repl1=/var/tmp/streamfile_repl1.$$
-typeset streamfile_repl2=/var/tmp/streamfile_repl2.$$
-typeset streamfile_repl3=/var/tmp/streamfile_repl3.$$
-
 function cleanup
 {
 	log_must $RM $streamfile_full
```

Comment:
When you merge with master you'll need to change the test script `$RM`->`rm`, etc.